frontend/packages/core/src/AppProvider/short-link-state-hydrator.tsx

```diff
@@ -103,7 +108,13 @@ const ShortLinkStateHydrator = ({
       {state.fromShortLink && (
         <Grid container direction="column" alignItems="flex-end">
           <StyledAlert severity="warning">
-            Loaded shared state. Any local changes will not be preserved.
+            <div style={{ display: "flex" }}>
+              Loaded shared state &quot;
+              <Link href={generateShortLinkRoute(window.location.origin, state.hash)}>
+                {state.hash}
+              </Link>
```

Comment:
Should we have an alternative state if `state.hash` is an empty string to prevent showing something like `Loaded shared state "". Any local changes will not be preserved.`?

Reply:
Good call out, modified it to handle if hash is empty and it'll just look like it was previously.